code/modules/food/food/snacks.dm

```diff
@@ -4354,7 +4308,7 @@
 	if (!flat_icon)
 		flat_icon = getFlatIcon(src)
 	var/icon/I = flat_icon
-	color = "#FFFFFF" //Some fruits use the color var. Reset this so it doesn't tint the batter
+	color = "#FFFFFF" //Some fruits use the color var. Reset this so it doesnt tint the batter
```

Comment:
`doesn't`

Reply:
oh i think this is reverting a correction from the big spell check PR which conflicted with this. it's literally just a code comment though. will address other stuff when you're done.

Reply:
Me posting review comments means I've finished the review.